travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
@dreispt 
Ok, I'm agree with you
This can speed up the merge and also to identify if you have bugs.

I will change it